### PR TITLE
feat(types): react-animated-modal

### PR DESCRIPTION
**Pull request by manishprivet**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`

### CODE REVIEW

File: types/react-animated-modal/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    | "zoomInRight"
+    | "zoomInUp"
+    | "hinge"
+    | "jackInTheBox"
+    | "rollIn"
+    | "bounce"
+    | "flash";
+
+export interface ModalProps {
+    closemodal: () => any;
```
Comment (peterblazejewicz) [inline types/react-animated-modal/index.d.ts:51]:
```suggestion
    closemodal: () => void;
```

File: types/react-animated-modal/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    visible: boolean;
+    type: AnimationTypes;
```
Comment (peterblazejewicz) [inline types/react-animated-modal/index.d.ts:53]:
```suggestion
    /** @default 'flipInX' */
    type?: AnimationTypes | undefined;
```
this is not reuqired, it takes default implemmentatio (not documented though):
https://github.com/morfsys/react-animated-modal/blob/master/src/index.js#L94

Comment (manishprivet) [reply types/react-animated-modal/index.d.ts:53]: Yup, thanks for the pick

**Review by peterblazejewicz — APPROVED** [review]
@manishprivet thx!
LGTM!

**Comment by manishprivet** [comment]
Ready to merge

**Pull request merged**